app/views/idv/phone_errors/failure.html.erb

```diff
@@ -26,10 +26,14 @@
               ),
             ) %>
       </p>
+      <%= t('idv.failure.phone.rate_limited.options_header') %>
+      <ul>
+        <% if @gpo_letter_available %>
+          <li><%= t('idv.failure.phone.rate_limited.option_verify_by_mail_html') %></li>
+        <% end %>
+      </ul>
```

Comment:
bullet to insert here: Cancel and start over again after **x hours**

Reply:
Added.